src/cljs/athens/events.cljs

```diff
@@ -1882,31 +1882,25 @@
         {:fx [[:dispatch [:remote/paste-verbatim uid text start value]]]}))))
 
 
-(defn link-unlinked-reference
-  "Ignores case. If title is `test`:
-  test 1     -> [[test 1]]
-  TEST 10    -> [[test 10]]
-  [[attest]] -> [[at[[test]]`"
-  [string title]
-  (let [ignore-case-title (re-pattern (str "(?i)" title))
-        new-str           (string/replace string ignore-case-title (str "[[" title "]]"))]
-    new-str))
-
-
 (reg-event-fx
   :unlinked-references/link
-  (fn [_ [_ block title]]
-    (let [{:block/keys [string uid]} block
-          new-str (link-unlinked-reference string title)]
-      {:dispatch [:transact [{:db/id [:block/uid uid] :block/string new-str}]]})))
+  (fn [_ [_ {:block/keys [string uid] :as block} title]]
+    (js/console.debug ":unlinked-references/link:" uid)
+    (if-let [local? (not (client/open?))]
+      (let [unlinked-references-link-event (common-events/build-unlinked-references-link -1 uid string title)
+            tx-data                        (resolver/resolve-event-to-tx @db/dsdb unlinked-references-link-event)]
+        (js/console.debug ":unlinked-references/link: local?" local?)
```

Comment:
This `js/console.debug` doesn't bring much value in this place, since we're already in a branch based on value of `local?`.
Though it makes sense to print it before we make a decision based on printed value.

Reply:
Removed the `js/console.debug`.

> Though it makes sense to print it before we make a decision based on printed value.
Can you clarify?

Reply:
Junius, what Alex means is, the console log is there to let us know what is the value of `local?` but with the above implementation, we only get this console log if `local?` results in `true` i.e we only see the console log if the event is local and won't be able to see it if this is a remote event. Here is a way we can implement this:

```
(let [local?  (not (client/open?))]
   (js/console.debug ":unlinked-references/link: local?" local?)  ;; Here we will know the value of `local?` in console log  for both true and false val of `local?`
   (if local?
      (do something)
      (else do something else)))
```

Reply:
That's why it's there. I should have asked earlier. I used `if-let` to the other events that I've done. I'll have to update those too.